src/masonite/commands/QueueFailedCommand.py

```diff
@@ -0,0 +1,39 @@
+"""New Key Command."""
+from cleo import Command
+from ..utils.filesystem import make_directory
+import os
+import pathlib
+import datetime
+
+
+class QueueFailedCommand(Command):
+    """
+    Creates a failed jobs table
+
+    queue:failed
+        {--d|--directory=database/migrations : Specifies the database connection if using database driver}
```

Comment:
Here shouldn't it be the directory of migrations instead of the connection ?
```suggestion
        {--d|--directory=database/migrations : Specifies the database migrations directory if using database driver}
```